frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-inventory-card.tsx

```diff
@@ -30,17 +31,30 @@ export const VMInventoryCard: React.FC<VMInventoryCardProps> = () => {
   // prefer vmi over vm if available (means: is running)
   const nicCount = vm ? getNetworks(vm).length : getVMINetworks(vmi).length;
   const disks = vm ? getDisks(vm) : getVMIDisks(vmi);
-  const diskCount = disks.filter((d) => d?.disk).length;
-  const cdromCount = disks.filter((d) => d?.cdrom).length;
+  const volumes = vm ? getVolumes(vm) : getVMIVolumes(vmi);
```

Comment:
why do these counts have to make up a number of disk in total?

I think it is a bit misleading. Especially when you have intersections between each. For example you can have CDROM environment variable.

Reply:
+1, removed the env counter